src/content/configuration/optimization.md

```diff
@@ -3,17 +3,22 @@ title: Optimization
 sort: 8
 contributors:
   - EugeneHlushko
+  - jeremenichelli
+related:
+  - title: 'webpack 4: Code Splitting, chunk graph and the splitChunks optimization'
+    url: https://medium.com/webpack/webpack-4-code-splitting-chunk-graph-and-the-splitchunks-optimization-be739a861366
 ---
 
-Optimization configuration allows you to control the webpack output optimization level.
+Since version 4 webpack runs optimizations for you depending on the `mode` set, but you can modify them indepentently too.
```

Comment:
Maybe say:
- ...depending on the  *chosen* `mode` or `mode` *selected*. Word *set* here is a bit confusing to me.
- ...but you can modify them indepentently too. -> All optimizations are available for manual configuration and overrides.